src/components/combo/themes/shared/combo.bootstrap.scss

```diff
@@ -128,4 +141,25 @@ $dropdown-theme: dropdown-theme.$bootstrap;
             color: inherit;
         }
     }
+
+    ::slotted([slot='suffix']),
+    [part|='toggle'] {
+        border-inline-end: .0625rem solid var(--disabled-border-color);
```

Comment:
Please, use the rem function directly to represent values. Seeing rem(1px) is much easier to understand than .0625rem. Also, the benefit of using the rem function that we have is that the value is bound to the value of the global base font size. If you have issues with using the function just interpolate the call `#{rem(1px)} solid var(--disabled-border-color)`